src/utils/client.ts

```diff
@@ -383,11 +384,14 @@ export default class Client {
       })
       .then((data: any) => {
         if (data === undefined) return Promise.resolve();
-        Object.entries<{ form_completed: boolean }>(data.sessions).forEach(
-          ([formKey, session]) =>
-            (initState.formSessions[formKey].form_completed =
-              session.form_completed)
-        );
+        Object.entries<{ form_completed: boolean }>(
+          data.completed_forms
+        ).forEach(([formKey, session]) => {
+          if (!initState.formSessions[formKey])
+            initState.formSessions[formKey] = {};
+          initState.formSessions[formKey].form_completed =
```

Comment:
Before resolving adding the guard above bug, login wasn't working properly because this would silently fail and stop execution of this 'thread' - i.e. setAuthId below was never called..